pkgs/top-level/all-packages.nix

```diff
@@ -10531,7 +10531,7 @@ with pkgs;
     spidermonkey = spidermonkey_1_8_5;
     python = python27;
     sphinx = python27Packages.sphinx;
-    erlang = erlangR16;
+    erlang = erlangR17;
```

Comment:
Can't it build against 18? Would be nice to not lock to particular versions as much as we can get away with it.